build.gradle

```diff
@@ -24,6 +24,7 @@ buildscript {
 
 plugins {
     id 'com.github.hierynomus.license' version '0.15.0' apply false
+	id 'com.github.ben-manes.versions' version '0.21.0'
```

Comment:
tabs => spaces?

Reply:
fyi, I meant all of your lines are tabs

Reply:
Right, fixed :)

Reply:
My fault. I've been using `tabs` instead of `spaces` in my Java code style settings. Since now changed to `spaces' for this project :)